mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/entity/attributes/Attribute.java

```diff
@@ -61,6 +71,8 @@ public enum Type {
     /**
      * Type of the attribute.
      */
+    // JsonIgnore added, otherwise type was serialized twice
+    @JsonIgnore
```

Comment:
Do we really need the field `type`? I am about to remove it completely. Could we rely on concrete subtype?

Reply:
We should not modify an existing model class.